packages/now-static-build/test/fixtures/10-gatsby/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
-    "gatsby": "^2.13.3",
+    "gatsby": "latest",
     "gatsby-image": "^2.2.4",
     "gatsby-plugin-manifest": "^2.2.1",
     "gatsby-plugin-offline": "^2.2.0",
```

Comment:
Should these other gatsby plugins be `latest` too?

Reply:
I'll update the other one too.